### PR TITLE
[walking_group] Adding option to re-select chair at resting point

### PR DESCRIPTION
**Pull request by cdondrup**
After yesterday's walking group it became apparent that the ability to select a different chair once the robot reach one is very important.

This PR adds this functionality.

Here is how it works:
- Send the robot to the next waypoint using the old admin interface:
  screenshot from 2016-04-05 10 22 01
- When it reaches the waypoint, once the "Rast" button
  screenshot from 2016-04-05 10 23 04
  is pressed, the robot goes into the resting mode. This means it shows the chair seclection as usual but when the control card is shown, the admin interface is slightly different:
  screenshot from 2016-04-05 10 24 35
- When "Stuhlauswahl" is selected it goes back to the chair selection. This can be triggered at any time even while it is still approaching a different chair.
- The resting mode ends once the "Weiter" button is pressed. This sends the robot off again and if the admin interface is triggered now, the old version will be shown:
  screenshot from 2016-04-05 10 22 01

All this switching of interfaces is necessary to prevent ambiguities when it comes to which chairs are to be selected.

Long story short: Once the therapists selected the first chair for a resting point, they can show the command card and then use "Stuhlauswahl" to change that chair at any point in time. When they send the robot to the next waypoint, this option vanishes until they select an initial chair for the new resting point. I hope that explains it well enough so @denisehe can explain it to the therapists tomorrow. I guess it is best to just let them try during one of the resting phases.

### CODE REVIEW

**Comment by denisehe** [comment]
Hello, okay!
Really cool!
Thanks for implementing this so quickliy :-D

**Comment by cdondrup** [comment]
@marc-hanheide the tests failed even though they shouldn't and when clicking details it wants log in information.  I think something might be broken.

**Comment by marc-hanheide** [comment]
retest this please

**Comment by marc-hanheide** [comment]
@cdondrup seems to have been some network troubles at UoL... I triggered a retest just now.

**Comment by marc-hanheide** [comment]
ok, does this need merging on Henry still?

**Comment by cdondrup** [comment]
Thank you! Yes it does still need merging and a re-build of the workspace since it changes one of the action definitions.
